x/ibc/20-transfer/module.go

```diff
@@ -1,31 +1,109 @@
 package transfer
 
 import (
+	"encoding/json"
+
 	"github.com/gorilla/mux"
 	"github.com/spf13/cobra"
 
+	abci "github.com/tendermint/tendermint/abci/types"
+
 	"github.com/cosmos/cosmos-sdk/client/context"
 	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
 	"github.com/cosmos/cosmos-sdk/x/ibc/20-transfer/client/cli"
 	"github.com/cosmos/cosmos-sdk/x/ibc/20-transfer/client/rest"
+	"github.com/cosmos/cosmos-sdk/x/ibc/20-transfer/types"
 )
 
+const (
+	ModuleName = types.SubModuleName
+)
+
+var (
+	_ module.AppModule      = AppModule{}
+	_ module.AppModuleBasic = AppModuleBasic{}
+)
+
+type AppModuleBasic struct{}
+
 // Name returns the IBC transfer ICS name
-func Name() string {
+func (AppModuleBasic) Name() string {
 	return SubModuleName
 }
 
-// RegisterRESTRoutes registers the REST routes for the IBC transfer
-func RegisterRESTRoutes(ctx context.CLIContext, rtr *mux.Router) {
+func (AppModuleBasic) RegisterCodec(cdc *codec.Codec) {
```

Comment:
nit, missing godoc comments in this file

Reply:
do we really need godoc here? these functions don't do anything interesting